setup.py

```diff
@@ -30,6 +30,38 @@ def package_data(pkg, roots):
     return {pkg: data}
 
 
+def is_requirement(line):
+    """
+    Return True if the requirement line is a package requirement;
+    that is, it is not blank, a comment, or editable.
+    """
+    # Remove whitespace at the start/end of the line
+    line = line.strip()
+
+    # Skip blank lines, comments, and editable installs
+    return not (
```

Comment:
Not that it's very important, but you probably want to do this [with a regexp match](https://docs.python.org/2/library/re.html) instead — no?